api/icd/icdv4/connection.go

```diff
@@ -45,6 +62,7 @@ type Uri struct {
 		CertificateBase64 string `json:"certificate_base64"`
 	} `json:"certificate"`
 	Database interface{} `json:"database"`
+	Bundle interface{}   `json:"bundle"`
```

Comment:
Do we need Bundle interface{} in type Uri struct because Secure   CassandraUri  `json:"secure"` (cassandra) is of type CassandraUri

Reply:
Correct.